gatorgrouper/utils/read_student_file.py

```diff
@@ -17,7 +17,7 @@ def read_student_file(filepath):
 
     # transform into desired output
     responses = list()
-    for record in csvdata:
+    for record in csvdata[1:]:
```

Comment:
Can you explain what this change means? Why does it need to be different? That information could be added to the PR description.

Reply:
This is due to the bad design of `read_student_file.py` and the original sample csv. If the input csv file has the header, that function will make the header name as the first list and there is no exception handling for it. I used `[1:]` to avoid collecting output of the first list which can be the header. However, this is not the solution for this bad design. @thomad74 is responsible for this file based on Issue https://github.com/GatorEducator/gatorgrouper/issues/184. There will be another PR to fix this bad design.